services/illiad.md

```diff
@@ -0,0 +1,58 @@
+
+#### Restart the IIS Manager
+
+1. Connect to lib-illsql.princeton.edu with an PRINCETON domain account that has admin rights on the server
+2. Click on the Windows icon and select the search icon.
+3. Search for Internet Information Services (IIS) Manager. 
+4. Right-click the IIS Manager and select "run as administrator".
+5. Browse to the "Default Web Site"
+6. Restart the IIS Service by selecting restart under "Manage Website section".
+
+
+##### Restart the ILLiad System Manager Service
+1. Connect to lib-illsql.princeton.edu with an PRINCETON domain account that has admin rights on the server
+2. Click on the Windows icon and select the search icon.
+3. Search for Services. 
+4. Right-click on Services and select "run as administrator".
+5. Browse to the services starting with "Illiad"
+6. Select the "Illiad System Manager" and restart it. 
+
+
+## Staff Clients
+Staff clients are part of a Windows image. When ILLiad has a new release, we must download new images and apply them to the client-side system manually. Staff users have a unique username/password stored in Illiad itself that they use to sign in to one of the three ILLiad clients: 
+
+* The "ILLiad" staff client users sign in to do requests. 
+* "Staff Manager" allows you to create and manage existing staff client users. Including reseting passwords. 
+* "Customization Manager" contains configuration settings for ILLiad as well as configuration settings for server-side add-ons.
+
+## Add-ons
+ILLiad has an add-on architecture implemented in [LUA](https://www.lua.org/). Add-ons can be added at the individual client level or on the server. Client add-ons are incorporated into our illiad client install in our staff workstation image and replicated on every client instance. Server add-ons are installed in a single place within the ILLiad server application. See available [add-ons](https://atlas-sys.atlassian.net/wiki/spaces/ILLiadAddons/pages/3149543/ILLiad+Addon+Directory).
+
+### Server Add-ons we use.
+* [ReCAP Add-On](https://github.com/PrincetonUniversityLibrary/illiad_scsb_addon) - Facilitates transactions with SCSB
+* ReShare Add-On Facilitates transactions with our ReShare system for borrowing and lending requests that are part of our Borrow Direct partnership. Check with PUL Resource Sharing for where to obtain the current version of this. 
+
+### Client Add-ons we use
+* [Alma NCIP](https://github.com/pulibrary/alma-ncip) - facilitates circulations transactions with Alma for both ILL borrowing and lending requests. 
+
+### Using the client for troubleshooting/account management
+We have several desktop machines that are available for remote desktop connection that will allow IT staff who don't use windows to utilize. Once connected you need actual illiad staff credentials to login into the various clients. 
+
+* lib-ares-cli1
+* lib-ares-cli2
+* lib-ares-cli3
```

Comment:
If these machines are used for Aeon, Ares, AND Illiad, I think a shared section of this document that says "all three applications use the same three Windows VMs. For historical reasons, they are called lib-ares-cli1/2/3." And eventually, maybe we should rename them, though that's not something we can do as part of this PR.